Onboarding/Sources/Onboarding/Services/Mocks/MockInterface.swift

```diff
@@ -0,0 +1,23 @@
+import Foundation
+
+protocol MockInterface: AnyObject {
+    func loadJSON(filename: String) -> [OnboardingSlideModel]
+}
+
+extension MockInterface {
+    func loadJSON(filename: String) -> [OnboardingSlideModel] {
+        guard let resourceUrl = Bundle.module.url(
+          forResource: "OnboardingResponse",
```

Comment:
Static load of OnboardingResponse

Reply:
this has been changed to use the filename to load the response